### PR TITLE
[Python] #37 simulate games and save the results

### PR DESCRIPTION
**Pull request by ths2o**

### CODE REVIEW

File: playground/house2.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import os
+import time
+from random import random
+from board import Board, Stone
+from referee import Referee
+from bot import Bot
+import util
+from repo.game import save_game_result
+from connect_db import DBConfig
+
```
Comment (chzlithoo) [inline playground/house2.py]: Please sort import orders

File: playground/house2.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        self.bot = Bot(self.board.config)
+        self.first_player_first_move = first_player_first_move
+        self.bot_alias = {
+            "alex": self.bot.alex_bot,
+            "tei": self.bot.tei_bot,
+            "jw": self.bot.alex_bot
+        }
+
+    @staticmethod
+    def player_input(color: str) -> Stone:
```
Comment (hyoungseok) [inline playground/house2.py]: seems better to remove unusing part

File: playground/house2.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        x = input("x: ")
+        y = input("y: ")
+        return Stone(x, y, color)
+
+    def play(self, render_every: bool = True):
```
Comment (hyoungseok) [inline playground/house2.py]: seems better to remove unusing part

File: playground/house2.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            print(f"{order} stone: {stone.x},{stone.y},{turn}")
+
+            game_info = {}
+            end_check = self.referee.end_check(self.board.log, self.board.config)
+            if end_check.is_end:
+                winner = "no one" if end_check.is_tie else order
+                print(f"{winner} wins")
+                self.board.print_winner(winner)
+
+                if save:
```
Comment (hyoungseok) [inline playground/house2.py]: maybe need additional condition to do not save tie game

File: playground/house2.py [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+                        game_info["black_player"] = p2
+                        game_info["white_player"] = p1
+
+                    game_info["first_move"] = self.first_player_first_move
+                    game_info["log"] = self.board.log
+
+                    engine = "mysql+pymysql://root:allmeat@35.233.179.226:3306/playground_log"
+                    db = DBConfig()
+                    session = db.setup_db_connection(engine)
+                    save_game_result(session, winner=game_info['winner'], board_config=game_info['config'],
```
Comment (hyoungseok) [inline playground/house2.py]: we use following convention for long argument input
Screen Shot 2020-03-09 at 8 36 58 PM

and use double quotation rather then single

File: playground/house2.py (hunk 1 above)
Comment (hyoungseok) [reply playground/house2.py]: for reference, my IDE recommends as follows

```
import os
import time
from random import random

import util
from board import Board
from bot import Bot
from connect_db import DBConfig
from referee import Referee
from repo.game import save_game_result
```

**Pull request merged**